### ISSUE TITLE
Some arguments are unrecognized.

### ISSUE THREAD

**Issue by leaves-slient** [YuweiYin/GTrans#2]
Hello,
I see your example training code, and i make new environment and pip install your code.
But i dont see the lr "my_inverse_sqrt" and the arguments(-encoder-block-layers --extract-encoder-hidden-states --decoder-block-layers) in your code.
So when training, the arguments above cannot be recoginzed.

**Comment by YuweiYin** [comment]
Thanks for raising the issue. The bash script of WMT-2014 En-De is outdated. We use "inverse_sqrt" as the value of --lr-scheduler. We have renamed "--encoder-block-layers" as "--encoder-group-layers" and removed the argument "--extract-encoder-hidden-states".

You can check or modify [transformer.py](https://github.com/YuweiYin/GTrans/blob/master/GTrans/fairseq/models/transformer.py#L1059) and [options.py](https://github.com/YuweiYin/GTrans/blob/master/GTrans/fairseq/options.py) for argument and hyper-parameter settings.

**Comment by leaves-slient** [comment]
Thanks for response. I think your work is wonderful, and i mainly want to use your code to make translaton such as WMT14 task, i see the transformer.py in github, but transformer.py takes no change compared to the default transformer.py in fairseq,&nbsp;whether the wrong version is uploaded.






------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "YuweiYin/GTrans"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2023年3月2日(星期四) 下午2:44
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [YuweiYin/GTrans] the lr "my_inverse_sqrt" and some arguments are unrecognized. (Issue #2)





 
Thanks for raising the issue. The bash script of WMT-2014 En-De is outdated. We use "inverse_sqrt" as the value of --lr-scheduler. We have renamed "--encoder-block-layers" as "--encoder-group-layers" and removed the argument "--extract-encoder-hidden-states".
 
You can check or modify transformer.py and options.py for argument and hyper-parameter settings.

**Comment by YuweiYin** [comment]
Thanks for your appreciation. We'll check the code and update it later. The implementation is easy: we pick up and fuse some hidden states of different layers, like in Figure 2 in our [paper](https://arxiv.org/pdf/2207.14467.pdf). The fusion function is defined in Equation 2, where $w_i^e$ denotes an individual parameter of the i^th layer in the encoder.

**Comment by YuweiYin** [comment]
Please refer to the GTrans model in [GTrans.py](https://github.com/YuweiYin/GTrans/blob/master/GTrans/fairseq/models/GTrans.py#L2514)
